src/components/Shell/Shell.tsx

```diff
@@ -135,18 +142,65 @@ export const Shell = ({ appNeedsUpdate, children, userPeerId }: ShellProps) => {
     document.title = title
   }, [title])
 
+  const enterFullscreen = () => {
+    const body: any = document.body
+    if (body.requestFullscreen) {
+      body.requestFullscreen()
+    } else if (body.webkitRequestFullscreen) {
+      body.webkitRequestFullscreen()
+    } else if (body.mozRequestFullScreen) {
+      body.mozRequestFullScreen()
+    } else if (body.msRequestFullscreen) {
+      body.msRequestFullscreen()
+    }
+  }
+
+  const exitFullscreen = () => {
+    const document: any = window.document
+    if (document.exitFullscreen) {
+      document.exitFullscreen()
+    } else if (document.webkitExitFullscreen) {
+      document.webkitExitFullscreen()
+    } else if (document.mozCancelFullScreen) {
+      document.mozCancelFullScreen()
+    } else if (document.msExitFullScreen) {
+      document.msExitFullScreen()
+    }
```

Comment:
While testing this PR, I noticed this error being thrown a few times:

```
Uncaught (in promise) TypeError: Not in fullscreen mode
    exitFullscreen Shell.tsx:161
    Shell Shell.tsx:177
```

It suggests that this function is getting called more often than it should. It might be worth adding some conditional logic around the call site to avoid invoking the function unnecessarily, or potentially wrapping the body of this function in a `try`/`catch` to silence the error if it is unavoidable.